src/test/e2e/00_use_cli_test.go

```diff
@@ -83,17 +83,21 @@ func TestUseCLI(t *testing.T) {
 
 	t.Run("zarf prepare find-images --kube-version", func(t *testing.T) {
 		t.Parallel()
+		controllerImageWithTag := "quay.io/jetstack/cert-manager-controller:v1.11.1"
+		controlImageWithSignature := "quay.io/jetstack/cert-manager-controller:sha256-4f1782c8316f34aae6b9ab823c3e6b7e6e4d92ec5dac21de6a17c3da44c364f1.sig"
+
 		// Test `zarf prepare find-images` on a chart that has a `kubeVersion` declaration greater than the default (v1.20.0)
 		stdOut, stdErr, err := e2e.Zarf("prepare", "find-images", "src/test/packages/00-kube-version-override")
-		require.Error(t, err, stdOut, stdErr)
-		require.Contains(t, stdErr, "Problem rendering the helm template for https://charts.jetstack.io/", "The kubeVersion declaration should prevent this from templating")
-		require.Contains(t, stdErr, "following charts had errors: [https://charts.jetstack.io/]", "Zarf should print an ending error message")
+
+		require.NoError(t, err, stdOut, stdErr)
+		require.Contains(t, stdOut, controllerImageWithTag, "The chart image should be found by Zarf")
+		require.Contains(t, stdOut, controlImageWithSignature, "The image signature should be found by Zarf")
 
 		// Test `zarf prepare find-images` with `--kube-version` specified and greater than the declared minimum (v1.21.0)
 		stdOut, stdErr, err = e2e.Zarf("prepare", "find-images", "--kube-version=v1.22.0", "src/test/packages/00-kube-version-override")
 		require.NoError(t, err, stdOut, stdErr)
```

Comment:
This should now check for an error and specify a kubeversion lower than the chart expects (to ensure that the --kubeversion flag still works)